src/9FC043B0.c

```diff
@@ -1,19 +1,119 @@
 #include "include_asm.h"
+#include "macros.h"
+#include "bbtypes.h"
+#include "libcrypto/aes.h"
 
-INCLUDE_ASM("asm/non_matchings/9FC043B0", recrypt_list_verify_ecc_sig);
+extern BbVirage2* virage2_offset;
 
-INCLUDE_ASM("asm/non_matchings/9FC043B0", recrypt_list_sign);
+void virage2_gen_public_key(u32* pubkeyOut);
 
-INCLUDE_ASM("asm/non_matchings/9FC043B0", recrypt_list_clear);
+s32 recrypt_list_verify_ecc_sig(RecryptList* list) {
+    BbEccPublicKey publicKey;
+    s32 size;
```

Comment:
```suggestion
    u32 size;
```
Unsigned probably makes more sense for a size quantity